src/components/Page/Toolbar/Toolbar.css

```diff
@@ -15,6 +15,14 @@
   padding: 0px;
 }
 
+div.left {
```

Comment:
This will apply to all divs. Please make it specific to the top toolbar div.

Reply:
ok then

Reply:
have a look now ☺️